controllers/applicationvolumereplication_controller.go

```diff
@@ -116,19 +115,28 @@ func (r *ApplicationVolumeReplicationReconciler) Reconcile(ctx context.Context,
 	if err := r.updateAVRStatus(ctx, avr, placementDecisions); err != nil {
 		logger.Error(err, "failed to update status")
 
-		return ctrl.Result{}, err
```

Comment:
I thought returning an error from the reconciler automatically requeues

Reply:
That's right: https://github.com/operator-framework/operator-sdk/issues/2898

Reply:
You are right, I think i changed all returns when looking at this function. We should possibly not return the err and just request for a requeue (the auto stack trace else is a pain in the logs).